collect_app/src/main/java/org/odk/collect/android/formmanagement/ServerFormDownloader.java

```diff
@@ -96,6 +96,8 @@ private void processOneForm(ServerFormDetails fd, OngoingWorkListener stateListe
                 FormMediaDownloader mediaDownloader = new FormMediaDownloader(formsRepository, formSource);
                 newAttachmentsDetected = mediaDownloader.download(fd, tempMediaPath, tempDir, stateListener);
             }
+
+            ServerFormDownloaderUseCases.INSTANCE.copySavedFileFromPreviousFormVersionIfExists(formsRepository, fd.getFormId(), tempMediaPath);
```

Comment:
You can use `@JvmStatic` on the method here to avoid the `.INSTANCE`.

Reply:
Done.